src/script/bitcoinconsensus.h

```diff
@@ -12,13 +12,13 @@
 #include <config/bitcoin-config.h>
   #if defined(_WIN32)
     #if defined(DLL_EXPORT)
-      #if defined(HAVE_FUNC_ATTRIBUTE_DLLEXPORT)
+      #if defined(HAVE_DLLEXPORT_ATTRIBUTE)
```

Comment:
We could also always assume `__declspec(dllexport)` to be present on windows? It seems something is very wrong with the compiler if it is not.

Reply:
> We could also always assume __declspec(dllexport) to be present on windows? It seems something is very wrong with the compiler if it is not.

Agree. Also, this is our internal build, so we're generally aware of what we support, and we wouldn't support such a broken compiler.

Reply:
In that case, the dllexport can be moved to a Windows-only check in configure, which errors if it fails.